File: Lunchtime/Controller/SetupPageSixthViewController.m
//
//  SetupPageFifthViewController.m
//  Lunchtime
//
//  Created by Willow Belle on 2015-11-13.
//  Copyright © 2015 Cosmic Labs. All rights reserved.
//

#import "SetupPageSixthViewController.h"

static NSString *kUserCreatedFlag = @"USER_CREATED";
static NSString *kSegueFromSetupFlow = @"segueFromSetupFlow";

@interface SetupPageSixthViewController ()

@property (nonatomic, weak) IBOutlet UIButton *doneButton;

@end

@implementation SetupPageSixthViewController

- (IBAction)doneButtonPressed:(UIButton *)sender {
    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
    [defaults setObject:kUserCreatedFlag forKey:kUserCreatedFlag];

    [self performSegueWithIdentifier:kSegueFromSetupFlow sender:self];
}

@end

File: Lunchtime/Controller/SetupPageSixthViewController.swift
//
//  SetupPageSixthViewController.swift
//  Lunchtime
//
//  Created by Willow Belle on 2015-11-18.
//  Copyright © 2015 Cosmic Labs. All rights reserved.
//

import UIKit

class SetupPageSixthViewController: UIViewController {

    // MARK: - Constants
    private struct Constants {
        static let UserCreatedFlag = "USER_CREATED"
        static let SegueFromSetupFlow = "segueFromSetupFlow"
    }

    // MARK: - Outlets
    @IBOutlet weak var doneButton: UIButton?

    // MARK: - Actions
    @IBAction func doneButtonPressed(sender: UIButton) {
        let defaults = NSUserDefaults.standardUserDefaults()
        defaults.setObject(Constants.UserCreatedFlag, forKey: Constants.UserCreatedFlag)
        performSegueWithIdentifier(Constants.SegueFromSetupFlow, sender: self)
    }
}
